.github/CHANGELOG.md

```diff
@@ -6,12 +6,17 @@
 
 <h3>Bug fixes</h3>
 
+* The `frontend` tests can now be run without having TensorFlow installed.
+  [(#639)](https://github.com/XanaduAI/strawberryfields/pull/639)
```

Comment:
Minor: not sure if this is a bug fix per se that we'd like to include in the changelog to notify users. :thinking:

If it affects the way TF is imported in `strawberryfields/backends/tfbackend/__init__.py`, then it is relevant. :ok_hand:

Reply:
Hmm, good call! I've removed the note from the breaking changes.